src/ClrPro.AzureFX.LocalCredentialBridge/Controllers/TokensBridgetController.cs

```diff
@@ -122,6 +122,18 @@ await CreateTokenFileAndAddToCache(localFileName, TimeSpan.FromSeconds(options.A
             });
     }
 
+    private static string GetLocalTokenPath(LocalCredentialBridgeOptions options)
```

Comment:
this service is using IConfiguration to bind it's settings - you can just set up environment variable injection in `Program.cs`

Reply:
Reverted